packages/css/src/components/error-message/README.md

```diff
@@ -0,0 +1,11 @@
+<!-- @license CC0-1.0 -->
+
+# Error Message
+
+Show an error message when there is a form field validation error.
+In the error message explain what went wrong and how to fix it.
```

Comment:
```suggestion
Explain in the error message what went wrong and how to fix it.
```

Reply:
Do you have a strong reason to want to change the order here? [I copied this sentence from gov.uk](https://design-system.service.gov.uk/components/error-message/), I'm kind of leaning towards the native speakers.

Reply:
No strong reason, just that the two sentences repeated a bit too much.